### DISCUSSION TITLE
tinyAVR 2-series "This works" list

### DISCUSSION THREAD

**Discussion by SpenceKonde** [SpenceKonde/megaTinyCore#317] [Future Development Discussion]
When you test things and they work, let us know.

**Comment by gcormier** [comment]
tone() works, but the sound is off by a hair - unsure if it's just a difference in internal oscillator calibration between tiny841 and tiny1624

**Reply by SpenceKonde** [reply]
Is it higher pitch on the 841? The 841 internal oscillator is consistently high at 5v, by like 3%

**Reply by gcormier** [reply]
Yup the tiny841 is higher.

**Reply by SpenceKonde** [reply]
cool, stay tuned on attinycore, I'm partway done with a tuner :-) 
If using Optiboot (obv, can't really do it otherwise cause there isn't a safe place to stash it that you can count on staying safe. - I guess I could have it save to EEPROM or something.

**Reply by gcormier** [reply]
I'll try to debug the avr2 this weekend. Maybe the clock skew is enough that the devices on the LIN bus won't respond, but it seems far fetched - I'm doing a 19200 baud rate. I could try a 19,800 on the 1624 to compensate for the tiny841's slightly faster clock.

**Reply by gcormier** [reply]
I'm wondering if it's related..

tone() sounds "scratchy". Like, the tone is mostly there. At first I wrote it off as nothing. But perhaps it is related to some sort of interrupt or timing issue?

My debugging continues but in case that gives you any sort of "aha!" moment figured I would share.
